Fix code samples in the README.md (#68)
wso2-ballerina/module-github: github4/Module.md

> The `getRepository` function gets a GitHub repository by passing the name of the repository and its owner in the format of "owner/repository".

Reviewer: ```suggestion
The `getRepository` remote function gets a GitHub repository by passing the name of the repository and its owner in the format of "owner/repository".
```

```ballerina
var repo = githubClient->getRepository("wso2-ballerina/module-github");
```

The response from `getRepository` is either a `Repository` object (if the request was successful) or an `error` (if the request was unsuccessful).
```ballerina
if (repo is github4:Repository) {
    io:println("Repository Details: ", repo);
} else {
    io:println("Error: ", repo);
}
```

> The `getIssueList` function gets a list of issues for a given repository by providing the `Repository` object or repository and owner name, state of the issue, and the number of records to read.

Reviewer: ```suggestion
The `getIssueList` remote function gets a list of issues for a given repository by providing the `Repository` object (repository and owners), state of the issue, and the number of records to read.
```

```ballerina
github4:Repository issueRepository = { owner: { login: "wso2" }, name: "carbon-apimgt" };
var issues = githubClient->getIssueList(issueRepository, github4:STATE_CLOSED, recordCount);
```
or
```ballerina
var issues = githubClient->getIssueList(("wso2", "carbon-apimgt"), github4:STATE_CLOSED, recordCount);
```

The response from `getIssueList` is either an `IssueList` object (if the request was successful) or an `error` (if the request was unsuccessful).

```ballerina
if (issues is github4:IssueList) {

>     io:println("Issue Details: ", issues);

Reviewer: Shall we change this to 
```suggestion
    io:println("Issue List: ", issues);
```
since we are demonstrating `createIssue` too?

} else {
    io:println("Error: ", issues);
}
```

The `createIssue` function creates a new issue in a given repository.

```ballerina

> var createdIssue = githubClient->createIssue (repositoryOwner, repositoryName, issueTitle, issueContent, labelList, assigneeList);

Reviewer: ```suggestion
var createdIssue = githubClient->createIssue(repositoryOwner, repositoryName, issueTitle, issueContent, labelList, assigneeList);
```
